src/components/orderstatuspopup.tsx

```diff
@@ -1,57 +1,39 @@
-import { useEffect, useState } from "react";
-import { Button } from "primereact/button";
 import { Column } from "primereact/column";
+import { FC } from "react";
+import { ORDER_STATUS_LIST } from "../constants/appConstants";
 import CustomDialog from "./common/customdialog";
 import CustomTable from "./common/customtable";
-import { ORDER_STATUS_LIST } from "../constants/appConstants";
-import CustomButton from "./Button";
 
-interface OrderStatus {
-  code: number;
-  description: string;
-}
+type Props = {
+  openDialog: boolean;
+  setOpenDialog: (a: boolean) => void;
+};
 
-const OrderStatusPopup = () => {
+const OrderStatusPopup: FC<Props> = ({ openDialog, setOpenDialog }) => {
   const HEADERS = [
     { field: "code", header: "Status Code" },
     { field: "description", header: "Description" },
   ];
 
-  const [orderStatusList, setOrderStatusList] = useState<OrderStatus[]>([]);
-  const [openDialog, setOpenDialog] = useState<boolean>(false);
-
-  useEffect(() => {
-    setOrderStatusList(ORDER_STATUS_LIST);
-  }, [ORDER_STATUS_LIST]);
-
   return (
-    <>
-      <CustomButton
-        label="open order status"
-        onClick={() => setOpenDialog(true)}
-        isTextButton
-        severity="secondary"
-        className="popup-button"
+    <CustomDialog
+      header="Order Status"
```

Comment:
Keep constant strings in constants file.
It will be easy to maintain in future.

Reply:
Done